src/test/java/org/isf/medicalsinventory/test/Tests.java

```diff
@@ -45,9 +45,9 @@
 import org.isf.ward.model.Ward;
 import org.isf.ward.service.WardIoOperationRepository;
 import org.isf.ward.test.TestWard;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
I deleted the comment about moving to JUnit 5 because it requires other changes.   Please revert this change.  Sorry and thank you.

Reply:
exactly i've encountered a problem and apparently it's due to the fact that we're using Junit4's SpringRunner.  okay let me revert the changes.